.github/workflows/npm-publish.yml

```diff
@@ -12,23 +12,18 @@ jobs:
     name: npm-publish
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@master
+      - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-          token: ${{ secrets.ADOBE_BOT_GITHUB_TOKEN }}
```

Comment:
token is no longer needed?

Reply:
I believe this line is what is causing the release action to fail. It does not appear that any of the actions across the SDKs use a token while calling the checkout action.

Reply:
Unless there is a different security configuration for this repo than the others. However, the `CI` action of this repo does not use a token during checkout.